### PR TITLE
added NCB

### PR DESCRIPTION
**Pull request by austinarchibald**

### CODE REVIEW

File: lib/sports_api/fetchers/scores/helpers/api_parser.rb [hunk 1]
```diff
@@ -89,9 +89,10 @@ def generate_competitors(event_json)
         competitor.abbreviation = competitor_json['team']['abbreviation']
         competitor.location = competitor_json['team']['location']
         competitor.id = competitor_json['team']['id']
+        competitor.is_active = competitor_json['team']['isActive']
         competitor.conference_id = competitor_json['team']['conferenceId']
         # NCF Conference API: http://site.api.espn.com/apis/site/v2/sports/football/college-football/scoreboard/conferences?groups=80%2C81
-        # NCB Conference API: Don't know yet
+        # NCB Conference API: http://site.api.espn.com/apis/site/v2/sports/basketball/mens-college-basketball/scoreboard/conferences?groups=50
```
Comment (austinarchibald) [inline lib/sports_api/fetchers/scores/helpers/api_parser.rb]: @MikeSilvis would appreciate any ideas on best approach to pulling in `competitor.conference_name` via these api urls, b/c that was my ultimate goal with these conference_ids anyway

**Comment by MikeSilvis** [comment]
@austinarchibald i don't know how i am just now seeing this. Can you please update the PR and fix the tests?

Looks like the tourney broke them

**Comment by austinarchibald** [comment]
Needed hoops to start up again to motivate me to get back on this :) ... rspec now passes all tests

**Comment by MikeSilvis** [comment]
Thanks man! Merged

**Pull request merged**